src/main/java/com/quantumsoft/qupathcloud/imageserver/CloudImageServerBuilder.java

```diff
@@ -55,8 +52,15 @@ public float supportLevel(URI uri, FileFormatInfo.ImageCheckType info, Class<?>
   }
 
   @Override
-  public ImageServer<BufferedImage> buildServer(URI uri) throws QuPathCloudException {
-    return new CloudImageServer(uri, cloudDao, Repository.INSTANCE.getDicomStore());
+  public UriImageSupport<BufferedImage> checkImageSupport(URI uri, String... args) {
```

Comment:
Add Doc stating why this new method is added and what the params mean.

Reply:
Ok.
API changes, previously supportLevel() itself was interface method, now it's wrapped into UriImageSupport.